modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,278 @@
+    spec:
+      serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
+      containers:
+        - name: provider-azure-installer
+          image: mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.1
+          imagePullPolicy: IfNotPresent
+          args:
+            - --endpoint=unix:///provider/azure.sock
+            - --construct-pem-chain=true
+            - --healthz-port=8989
+            - --healthz-path=/healthz
+            - --healthz-timeout=5s
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8989
+            failureThreshold: 3
+            initialDelaySeconds: 5
+            timeoutSeconds: 10
+            periodSeconds: 30
+          resources:
+            requests:
+              cpu: 50m
+              memory: 100Mi
+            limits:
+              cpu: 50m
+              memory: 100Mi
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: "/provider"
+              name: providervol
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: NotIn
+                values:
+                - virtual-kubelet
+      volumes:
+        - name: providervol
+          hostPath:
+            path: "/var/run/secrets-store-csi-providers"
+      tolerations:
+      - operator: Exists
+      nodeSelector:
+        kubernetes.io/os: linux
+----
++
+[NOTE]
+====
+This configuration is modified from the configuration provided in the link:https://azure.github.io/secrets-store-csi-driver-provider-azure/docs/getting-started/installation/[Azure documentation] so that it works properly with {product-title}.
+====
++
+////
+TODO: Modified the above YAML with the following:
+
+- Added the ClusterRole and ClusterRoleBinding
+- Added namespace to the serviceaccount and daemonset
+- Changed securityContext to privileged on the container
+
+See if all of these are correct or not. Removed the "oc adm policy add-scc-to-use" step
+////
+
+.. Create the provider resources by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f azure-provider.yaml
+----
+
+. Create a secret provider class to define your secrets store provider:
+
+.. Create a YAML file that defines the `SecretProviderClass` object:
++
+.Example `secret-provider-class-azure.yaml`
+[source,yaml]
+----
+apiVersion: secrets-store.csi.x-k8s.io/v1
+kind: SecretProviderClass
+metadata:
+  name: my-azure-provider                 <1>
+  namespace: my-namespace                 <2>
+spec:
+  provider: azure                         <3>
+  parameters:                             <4>
+    usePodIdentity: "false"
+    useVMManagedIdentity: "false"
+    userAssignedIdentityID: ""
+    keyvaultName: "kvname"
+    objects: |
+      array:
+        - |
+          objectName: tls-cert
```

Comment:
Can we mention like to look more simple or easier? 
          objectName: secret1
          objectType: secret

Reply:
Yep I can do that, will update!